samples/bluetooth/radio_coex_3wire/src/main.c

```diff
@@ -263,8 +266,83 @@ static void console_print_thread(void)
 	}
 }
 
+static int set_prio(void)
+{
+	int err = 0;
+	struct net_buf *buf;
+	sdc_hci_cmd_vs_coex_priority_config_t *cmd_params;
+
+	/* All bluetooth roles except SLAVE will use a low priority. */
+	for (int role = SDC_HCI_VS_COEX_BT_ROLE_ADVERTISER;
+	     role < SDC_HCI_VS_COEX_BT_ROLE_CENTRAL;
+	     role++) {
+		buf = bt_hci_cmd_create(SDC_HCI_OPCODE_CMD_VS_COEX_PRIORITY_CONFIG,
+					sizeof(*cmd_params));
+		if (!buf) {
+			printk("Could not allocate command buffer\n");
+			return -ENOMEM;
+		}
+		cmd_params = net_buf_add(buf, sizeof(*cmd_params));
+		cmd_params->role = role;
+		cmd_params->priority = MPSL_COEX_PRIORITY_LOW;
+		cmd_params->escalation_threshold = MPSL_COEX_ESCALATION_THRESHOLD_OFF;
+		err |= bt_hci_cmd_send_sync(SDC_HCI_OPCODE_CMD_VS_COEX_PRIORITY_CONFIG, buf, NULL);
```

Comment:
```suggestion
		err = bt_hci_cmd_send_sync(SDC_HCI_OPCODE_CMD_VS_COEX_PRIORITY_CONFIG, buf, NULL);
```
Is there is any reason for using `|=` here ?

Reply:
I don't think there is any, could you fix it @AutumnGul ?

Reply:
done